app/models/neighbourhood.rb

```diff
@@ -20,6 +20,8 @@ class Neighbourhood < ApplicationRecord
            source: :partners,
            class_name: 'Partner'
 
+  # validates :unit_code_value, presence: true, uniqueness: true
```

Comment:
should this have replaced the line below?

Reply:
OH LORD NO! if you put this line in the test suite goes into FULL FIRE mode. and then we'd have to go through the entire test suite and make it so our factories produce valid models.

this would be one of those "our tests could be better" but "this would take a lot of work" examples.

but it SHOULD be a constraint.